### PR TITLE
docs: readme cleanup

### PR DESCRIPTION
**Pull request by TimoGlastra**
Some cleanup updates to the README.

I've moved some sections over to the docs, we should probably also move the left-over parts to the docs (like installation for Action menu package) so it's not scattered anymore.

Will open a PR in docs soon also with migration etc.. and all the changes in 0.5.0

### CODE REVIEW

File: packages/tenants/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
     height="250px"
   />
 </p>
-<h1 align="center"><b>Credo - Tenant Module</b></h1>
+<h1 align="center"><b>Credo Tenant Module</b></h1>
```
Comment (genaris) [inline packages/tenants/README.md:9]: Is it 'Tenant' or 'Tenants'?

**Review by genaris — APPROVED** [review]

File: packages/tenants/README.md (hunk 1 above)
Comment (TimoGlastra) [reply packages/tenants/README.md:9]: good question :)

Comment (TimoGlastra) [reply packages/tenants/README.md:9]: probably tenants

Comment (TimoGlastra) [inline packages/tenants/README.md:9]:
```suggestion
<h1 align="center"><b>Credo Tenants Module</b></h1>
```

Comment (genaris) [reply packages/tenants/README.md]: I like those auto-suggestions. It remembers me the auto-passes in football 😄

**Pull request merged**